pydcs_extensions/f104/f104.py

```diff
@@ -1535,11 +175,9 @@ class Pylon6:
             6,
             Weapons.TER_9A_with_3_x_BDU_33___25lb_Practice_Bomb_LD,
         )
-
-    class Pylon7:
-        AIM_9M_Sidewinder_IR_AAM = (7, Weapons.AIM_9M_Sidewinder_IR_AAM)
-        AIM_9L_Sidewinder_IR_AAM = (7, Weapons.AIM_9L_Sidewinder_IR_AAM)
-        AIM_9B_Sidewinder_IR_AAM = (7, Weapons.AIM_9B_Sidewinder_IR_AAM)
+        LAU_105_2_AIM_9P5 = (6, Weapons.LAU_105_2_AIM_9P5)
+        LAU_105_2_AIM_9L = (6, Weapons.LAU_105_2_AIM_9L)
+        LAU_105_2_AIM_9P = (6, Weapons.LAU_105_with_2_x_AIM_9P_Sidewinder_IR_AAM)
```

Comment:
I messed this up, which is what I suspected after looking at @MetalStormGhost's PR #2189